python/ray/exceptions.py

```diff
@@ -133,17 +135,37 @@ def __str__(self):
         """Format a RayTaskError as a string."""
         lines = self.traceback_str.strip().split("\n")
         out = []
+        # We try not to print the error line if it is in worker
+        # because that contains the internal implemenation of Ray
+        # which is unnecessary for users.
         in_worker = False
         for line in lines:
             if line.startswith("Traceback "):
                 out.append(f"{colorama.Fore.CYAN}"
                            f"{self.proctitle}"
                            f"{colorama.Fore.RESET} "
-                           f"(pid={self.pid}, ip={self.ip})")
+                           f"(pid={self.pid}, ip={self.ip}"
+                           f", label={self.label})")
```

Comment:
printing ray::bar() here isn't useful, since it's already present in the highlighted part of the exception message. Please remove this for tasks.